src/scalper.ts

```diff
@@ -136,7 +133,7 @@ async function scalperPerp() {
         Math.abs(hedgeDeltaClip),
         { orderType: 'limit'},
       );
-      // Check if any size left
+      // Check if any size in theory left
```

Comment:
Explain what in theory means here

Reply:
We're sending a limit order with a price that should fill it completely but we do not have a guarantee of that. So we need to run a check to see what qty actually gets filled before cancelling and recalcing orders